tests/manage/z_cluster/cluster_expansion/conftest.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework import config
+from ocs_ci.ocs.node import (
+    get_worker_nodes,
+    add_new_node_and_label_it,
+    add_new_node_and_label_upi,
+)
+from ocs_ci.ocs import machine as machine_utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def add_nodes():
+    """
+    Test for adding worker nodes to the cluster while IOs
+    """
+
+    def factory(ocs_nodes=None, node_count=3):
```

Comment:
why not have ocs_nodes set as False by default instead?  I think that you can then omit the ocs_label variable defined on line 24 as you will have boolean value all the time, so you can directly use ocs_nodes variable on the places where you are using ocs_label variable.

Reply:
done